### DISCUSSION TITLE
Prefill creating item form with query params

### DISCUSSION THREAD

**Discussion by laekem34** [directus/directus#10416] [Archive]
Hello ! 

It could be a great feature if we could pre complete some item form field  with url query param
ex: `http://localhost:8055/admin/content/Releve/+?Value=30&patient=1`

Here Value is a float and parent and item in another collection in O2M (1 is the id)
We could use button for redirect item to creating precompleted another one
![image](https://user-images.githubusercontent.com/21008981/145403327-eb1a65c9-af08-4349-ad54-30bb988a8e6f.png)

**Comment by benhaynes** [comment]
Great idea! Ironically this was posted last week... we can consolidate the conversation there:
https://github.com/directus/directus/discussions/10232
